apps/broomva/app/globals.css

```
@@ -105,6 +105,35 @@
   --ag-shadow-glow-blue: var(--bv-shadow-feature);
   --ag-shadow-glow-accent: var(--app-shadow-glow-accent);
 
+  /* --- Elevation additions (BRO-2184): the named states that previously lived as arbitrary
+         shadow-[…] classes. Derived from the foundation where a token exists; the glows keep
+         their oklch values because the foundation defines no per-status glow. --- */
+  --ag-shadow-glow-soft: 0 0 12px oklch(0.60 0.12 260 / 0.08);          /* resting glass glow */
```

Stylelint (17.14.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)